.env.example

```
@@ -19,14 +22,18 @@ LOG_STACK=single
 LOG_DEPRECATIONS_CHANNEL=null
 LOG_LEVEL=debug
 
-DB_CONNECTION=mysql
+DB_CONNECTION=sqlite
+#DB_CONNECTION=mysql
 DB_HOST=127.0.0.1
+#DB_HOST=host.docker.internal
 DB_PORT=3306
-DB_DATABASE=laravel
+#DB_DATABASE=laravel
+DB_DATABASE=database
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DB_DATABASE key should go before the DB_HOST key

(UnorderedKey)